ddl/ddl_api.go

```diff
@@ -897,6 +904,14 @@ func (d *ddl) AlterTable(ctx context.Context, ident ast.Ident, specs []*ast.Alte
 	return nil
 }
 
+func (d *ddl) RebaseAutoID(ctx context.Context, ident ast.Ident, newBase int64) error {
+	t, err := d.infoHandle.Get().TableByName(ident.Schema, ident.Name)
+	if err != nil {
+		return errors.Trace(infoschema.ErrTableNotExists.GenByArgs(ident.Schema, ident.Name))
+	}
+	return t.RebaseAutoID(ctx, newBase, true)
```

Comment:
We'd better clear all TiDBs' cache.
So you can add a DDL Job likes https://github.com/pingcap/tidb/blob/master/ddl/ddl_api.go#L72
Then you can clear cache when we load the schema.

Reply:
Done. PTAL @zimulala